server/index.js

```diff
@@ -71,15 +71,40 @@ io.on('connection', (socket) => {
     });
 
      // received signal to start a session from the host of a room, emit redirect signal to all guests and host
-     socket.on('I am emitted from an imported socket!', () => {
-        console.log('successfully received emit from imported socket in swiping page')
+     socket.on('initialize_room', ({roomId, numGuests, movies}) => {
+        console.log('INITIALIZE ROOM:');
+        console.log(roomId);
+        console.log(numGuests);
+        initializeRoom(roomId, numGuests, movies).then((result) => {
+            console.log(result);
+        }).catch((error) => {
+            console.log('on initializing room, there was an error in redis');
+            console.log(error);
+        });
     });
 
     //received signal that no match was made
     socket.on('noMatch', (room) => {
         console.log('No Match signal received')
         io.to(room).emit('noMatchRedirect')
     })
+    socket.on('like_event', ({roomId, movieId, movieData}) => {
+        console.log('like-event')
+        console.log(movieId, movieData);
+        likeEvent(roomId, movieId).then((result) => {
+            console.log(result);
+            if (result===0) {
+                // initiate match page
+                io.to(roomId).emit('matchRedirect', { matchedMovieId: movieId, matchedMovieData: movieData });
+            }
+            else if (result===1) {
+                // initiate end of movies
+            }
+        }).catch((error) => {
+            console.log('error on like event to redis');
```

Comment:
I think we can combine these log statements as well.

Reply:
Sounds good!